pkg/component/component.go

```diff
@@ -96,6 +103,11 @@ func (r *RuntimeSpecs) ToComponents(policy map[string]interface{}) ([]Component,
 			"major":    r.platform.Major,
 			"minor":    r.platform.Minor,
 		},
+		"user": map[string]interface{}{
```

Comment:
Out of curiosity, what are we still using the transpiler for if we aren't doing config transformations anymore? Do we just need to do follow up surgery to completely remove it or does it need to stick around as an implementation detail?

Reply:
We need to do a cleanup to remove the transpiler rules and steps (neither are no longer used in V2). But the transpiler is still required as it performs all the variable substitution inside of the configuration for the dynamic inputs.

Reply:
I have cleaned up the transpiler rules and steps in the latest commit.